### PR TITLE
min master fix to avoid es split brain

### PR DESCRIPTION
**Pull request by jcantrill**
cc @portante

### CODE REVIEW

File: deployer/templates/es.yaml [hunk 1]
```diff
@@ -147,9 +150,13 @@ objects:
     description: "Numeric ID of a supplemental group for storage volume access."
     name: STORAGE_GROUP
     value: ${STORAGE_GROUP_DEFAULT}
+  -
+    name: MIN_MASTERS
```
Comment (ewolinetz) [inline deployer/templates/es.yaml]: Can we get a `description` on here?

**Comment by richm** [comment]
fyi, you don't need bc:
```
 +  es_min_masters=$(expr $es_cluster_size / 2 + 1)
```
expr is a bourne shell built-in

File: deployer/templates/es.yaml [hunk 2]
```diff
@@ -147,9 +150,14 @@ objects:
     description: "Numeric ID of a supplemental group for storage volume access."
     name: STORAGE_GROUP
     value: ${STORAGE_GROUP_DEFAULT}
+  -
+    description: "The mimimum number of masters that must be present in the ES cluster for the master election process to complete"
```
Comment (portante) [inline deployer/templates/es.yaml]: If we can, tying it back to the actual Elasticsearch parameter name it lands in would be really great, too.

**Comment by jcantrill** [comment]
@richm good to know.  Started to modify and was reminded this is on 1.4.  Going to pass on fixing as it works and will not be used going forward.

**Comment by ewolinetz** [comment]
for the sake of prosperity -- we missed adding an upgrade step/check in scripts/upgrade.sh for the DC patch

**Pull request merged**